src/crystal/system/win32/iocp.cr

```diff
@@ -110,8 +111,16 @@ module Crystal::IOCP
       pointerof(@overlapped)
     end
 
+    def internal_high
+      @overlapped.internalHigh
+    end
+
+    def internal_high=(value)
+      @overlapped.internalHigh = value
+    end
```

Comment:
**question:** Why expose these methods on `OverlappedOperation`? We could just access them directly through `@overlapped`.